eng/native/build-commons.sh

```diff
@@ -457,6 +457,22 @@ if [[ "$__PortableBuild" == 0 ]]; then
     __CommonMSBuildArgs="$__CommonMSBuildArgs /p:PortableBuild=false"
 fi
 
+if [[ "$__BuildArch" == wasm ]]; then
```

Comment:
Is this intentional change? I do not see how it is relevant to the rest.

Reply:
This was copied from build-native.sh when I had other libs embedded using QCall mechanism. 
We will need this to embed libs that use SSL stuff. I will remove this for now.

Reply:
Turns out we still need this. Since we build all the native libraries in clr partition. 

It is possible to exclude libs that are not yet linked from the build, but it would be extra conditions that soon will not be needed.

I will check if this can be moved to a single place, but it could end up being this place.

Reply:
Turns out most of this is not needed. We only need to make sure that FEATURE_DISTRO_AGNOSTIC_SSL is set if the build is portable.